src/documentation/io/built-in.md

```diff
@@ -56,6 +60,7 @@ Consult the [Programming Guide I/O section]({{site.baseurl }}/documentation/prog
 <tr>
   <td>Python</td>
   <td>
+    <p>Beam Python supports GCS and local filesystems out of the box. Support for HDFS is in progress (<a href="https://issues.apache.org/jira/browse/BEAM-3099">BEAM-3099</a>).</p>
```

Comment:
Apache HDFS
Should this second part be added to the table below instead?

Reply:
Good point, moved.